.github/workflows/agent-validation.yml

```
@@ -0,0 +1,86 @@
+
+      - name: Validate Agent Changes
+        env:
+          PR_TITLE: ${{ env.PR_TITLE }}
+          BASE_REF: ${{ env.BASE_REF }}
+          HEAD_REF: ${{ env.HEAD_REF }}
+        run: |
+          set -euo pipefail
+          echo "PR_TITLE: $PR_TITLE"
+          echo "BASE_REF:  $BASE_REF"
+          echo "HEAD_REF:  $HEAD_REF"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)